### ISSUE TITLE
Piston object

### ISSUE THREAD

**Issue by RenatoGeh** [MarvellousSoft/MarvInc#67]
Create an object piston that lets you move blocks. Uses console to set how many blocks the piston's handle moves.

A piston could also have a strength attribute, where the number of blocks it can move is directly related to its strength (e.g. strength=2 means it can move up to two blocks at the same time).

**Comment by yancouto** [comment]
Can you give an example on a level?

**Comment by RenatoGeh** [comment]
A path is blocked by walls. You could use a set of pistons to clear the path. But doing so could eventually lead to another path being blocked. A maze where pistons can be used to alter the maze itself is an example of this use.

**Comment by rilifon** [comment]
@RenatoGeh @yancouto Are we still making this?

**Comment by RenatoGeh** [comment]
I say postpone it. We can make this happen once we have a stable 1.0.

**Comment by yancouto** [comment]
Agreed.

**Comment by rilifon** [comment]
but if we have a stable 1.0, shouldn't we have all puzzles done?

but I agree this is not a priority

**Comment by RenatoGeh** [comment]
Then pistons will be made available in the next DLC: Pistons & Sewer Tiles.
